sv2-rpi.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/sh
+
+# This program utilizes cargo-cross to automate the generation of a tarball package distribution of
+# SRI releases targeting the Rasperry Pi OS (32 and 64 bits).
+
+print_help() {
+  echo "StratumV2 Reference Implementation (SRI) distribution for Raspberry Pi OS (32 and 64 bits)"
+  echo ""
+  echo "Usage:"
+  echo "$ ./sv2-rpi.sh <role> <bits>"
+  echo ""
+  echo "Available options for <role> are:"
+  echo "jd_client"
+  echo "jd_server"
+  echo "mining_device"
+  echo "mining_proxy_sv2"
+  echo "pool_sv2"
+  echo "translator_v2"
+  echo ""
+  echo "Available options for <bits> are:"
+  echo "32"
+  echo "64"
+  echo ""
+  echo "For example:"
+  echo "$ ./sv2-rpi.sh translator_sv2 64"
+}
+
+build(){
+  cross build -p $PKG --target $TARGET --release
+}
+
+clean_pkg(){
+  rm -rf pkg
+}
+
+clean_tar(){
+  rm -rf tar/$TARBALL
+}
+
+pkg(){
+  clean_pkg
+  mkdir -p pkg/$PKG
+  mkdir -p pkg/$PKG/bin
+  mkdir -p pkg/$PKG/etc/sri
+  mkdir -p tar
+}
+
+tarball(){
+  TARBALL=$PKG-$TARGET.tar.gz
+  clean_tar
+  pushd pkg
+  tar -cvf ../tar/$TARBALL .
+  popd
+  echo "tar/$TARBALL created with success."
+}
+
+jd_client(){
+    build
+    pkg
+
+    install target/$TARGET/release/jd_client pkg/$PKG/bin
+    # install roles/jd-client/config-examples/jdc-config-hosted-example.toml pkg/$PKG/etc/sri
+    # install roles/jd-client/config-examples/jdc-config-local-example.toml pkg/$PKG/etc/sri
```

Comment:
@Fi3 the config files seem to have wrong names on [`roles/jd-client/config-examples`](https://github.com/stratum-mining/stratum/tree/dev/roles/jd-client/config-examples)

Reply:
solved via https://github.com/stratum-mining/stratum/pull/684/commits/f9ccaae6dd93d702c40b307c053fb34644515ce9